applications/FluidDynamicsApplication/python_scripts/fluid_rve_analysis.py

```diff
@@ -0,0 +1,203 @@
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication
+from KratosMultiphysics.FluidDynamicsApplication.fluid_dynamics_analysis import FluidDynamicsAnalysis
+
+# Importing other libraries
+import math
+
+class FluidRVEAnalysis(FluidDynamicsAnalysis):
+    def __init__(self, model, project_parameters):
+
```

Comment:
We should add a validation of the user `rve_settings` in here. See how we do so in the solvers. Long story short the idea would be to hardcode a `KratosMultiphysics.Parameters` object containing the expected settings with default values and then call the input `rve_settings` against these by doing
```py
input_rve_settings = project_parameters["rve_settings"] # take the rve_settings from the input
default_settings = self.GetDefaultParameters() # this method returns the rve_settings we expect
input_rve_settings.ValidateAndAssignDefaults(default_settings) # this method validates the input ones with it
```

Reply:
Fixed this way:
class FluidDynamicsAnalysisRVE(FluidDynamicsAnalysis):

    @classmethod
    def GetDefaultParameters(cls):

        ##settings string in json format
        default_settings = KratosMultiphysics.Parameters("""
        {
            "boundary_mp_name": "unknown_name",
            "averaging_mp_name": "FluidModelPart",
            "print_rve_post": false,
            "fix_pressure": true,
            "jump_XX": 0,
            "jump_XY": 1e-6,
            "jump_YY": 0,
            "jump_XZ": 1e-6,
            "jump_ZZ": 0,
            "jump_YZ": 1e-6
        }""")

        return default_settings
    
    def __init__(self, model, project_parameters):

        # Check validity of RVE settings
        input_rve_settings = project_parameters["rve_settings"]
        default_rve_settings = self.GetDefaultParameters()
        input_rve_settings.ValidateAndAssignDefaults(default_rve_settings)


The `print_rve_post` field will be deleted when I fix the output settings

Reply:
OK. This was precisely what I had in mind. My unique comment would be to rename the `GetDefaultParameters` to `GetFluidRVEDefaultParameters` to make clear that these are the defaults of this portion of the input (not of the entire analysis stage).

Reply:
Done.